db/seeds/evaluatieonderzoek/questionnaires/evaluatieonderzoek.rb

```diff
@@ -1,85 +1,945 @@
+                       show_otherwise: false,
+                       title: 'Kunt u kort de belangrijkste taken noemen die u uitvoert in samenwerking met de afdelingen ‘dienst werk en inkomen’ van gemeenten in uw regio?',
+                       options: [{ title: 'Niet van toepassing', hides_questions: %i[v72_1] }],
+                       tooltip: 'In sommige gemeenten wordt dit anders genoemd, bijv. sociaal domein, participatie: het gaat hier om de afdeling verantwoordelijk voor onder andere arbeidsparticipatie.'
+                     }, {
+                       id: :v72_1,
+                       hidden: true,
+                       type: :textarea,
+                       title: '',
+                       placeholder: 'Vul hier uw antwoord in'
+                     }, {
+                       section_start: '',
+                       type: :raw,
+                       content: '<div class="divider"></div><p class="flow-text"><strong>Vooruitblik en Decentralisatie-uitkering</strong></p>',
+                     }, {
+                       id: :v73,
+                       type: :textarea,
+                       title: 'Heeft u nog advies of opmerkingen voor landelijk beleid om uw <strong>dagelijkse werkzaamheden</strong> te ondersteunen?',
+                       placeholder: 'Vul hier uw antwoord in'
+                     }, {
+                       id: :v74,
+                       type: :textarea,
+                       title: 'Heeft u nog advies of opmerkingen voor landelijk beleid om de <strong>samenwerking in uw regio</strong> verder te verbeteren?',
+                       placeholder: 'Vul hier uw antwoord in'
+                     }, {
+                       id: :v75,
+                       type: :textarea,
+                       title: 'Heeft u nog advies of opmerkingen voor landelijk beleid wat betreft de <strong>verdeling van financiële middelen</strong>?',
+                       placeholder: 'Vul hier uw antwoord in'
+                     }, {
+                       id: :v76,
+                       hidden: true,
+                       type: :radio,
+                       title: 'Momenteel wordt een deel van het regiobudget in de vorm van een specifieke uitkering (geoormerkt geld) uitgekeerd.<br>Is het volgens u mogelijk om deze specifieke uitkering van het regiobudget om te zetten in een decentralisatie-uitkering (niet geoormerkt)?',
+                       options: [
+                         { title: 'Ja' },
+                         { title: 'Nee' },
+                         { title: 'Weet ik niet' }
+                       ],
+                       show_otherwise: false,
+                       tooltip: 'Een deel van het regionaal budget voor VSV en kwetsbare jongeren komt via de contactgemeente naar de regio in de vorm van een specifieke uitkering met een specifiek doel (geoormerkt). Bij een decentralisatie-uitkering komt het bij de contactgemeente binnen \'op de grote hoop\', via het Gemeentefonds.'
```

Comment:
regiobudget vs. regionaal budget?

Reply:
willen we hier dan ook utf8 quotes? (muggenzifterig)

Reply:
regionaal budget kwam vaker voor en regiobudget kwam alleen in deze vraag voor, dus heb regiobudget hier aangepast naar regionaal budget (en ook de wijziging als suggestie doorgevoerd in de google doc).

Reply:
Heb er nu utf-8 quotes van gemaakt.